src/app/user/adapter/UserFileAdapter.js

```diff
@@ -102,10 +102,10 @@ const authenticate = async (username, password) => {
 
   if (!user) return null;
   const request = promisify(crypto.pbkdf2);
-  const userPasswordPolicyEntity = await userEntity.getUserPasswordPolicy();
+  const userPasswordPolicyEntity = await user.getUserPasswordPolicy();
```

Comment:
why was 'userEntity' renamed to 'user'?

Reply:
this was changed because user entity doesn't exist and is looking at user in the codebase so I changed to user instead of entityuser